ramls/schemas/jobExecution.json

```diff
@@ -0,0 +1,61 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Job Execution Schema",
+  "type": "object",
+  "properties": {
+    "id": {
+      "description": "Unique identifier",
+      "$ref": "../../ramls/raml-util/schemas/uuid.schema"
+    },
+    "hrId": {
+      "description": "Human readable id",
+      "type": "string"
+    },
+    "exportedFiles": {
+      "description": "List of files that should be exported",
+      "type": "array",
+      "uniqueItems": true,
+      "items": {
+        "type": "object",
+        "$ref": "exportedFile.json"
+      }
+    },
+    "jobProfile": {
```

Comment:
Can we name it the same way it is named in ui-data-import? [jobProfileInfo](https://github.com/folio-org/data-import-raml-storage/blob/master/schemas/mod-source-record-manager/jobExecution.json#L27), so UI should be able to be reused both for data-export and data-import.

Reply:
Yes, sure

Reply:
Thanks. By the way, will the `name` field be added to the `jobProfileInfo`? I see for data-import there is a such and it is used on the UI. It would be great for our case to have the same.

```
jobProfileInfo: {
  id: '32fafcc3-f582-493d-88b0-3c538480cd83',
  name: 'Standard BIB profile',
},
```

Reply:
Okay, will add it now